src/hooks/useReports.ts

```diff
@@ -16,126 +17,93 @@ export interface ReportTransactionData {
 const useReports = (ownerId?: string, gateway?: AoGateway) => {
   const arIOReadSDK = useGlobalState((state) => state.arIOReadSDK);
 
-  const startEpoch = useGlobalState((state) => state.currentEpoch);
-
   const observerAddress = gateway?.observerAddress;
   const gatewayStart = gateway?.startTimestamp;
 
-  const queryResults = useInfiniteQuery({
+  const { data: epochs } = useEpochs();
+
+
+  const queryResults = useQuery({
     queryKey: ['reports', ownerId],
-    queryFn: async ({ pageParam }) => {
+    queryFn: async () => {
       if (
         !arIOReadSDK ||
-        startEpoch === undefined ||
+        epochs === undefined ||
```

Comment:
epochs can also be null i think (or rather useQuery data can be null) so maybe !epochs would be better here to cover both?

Reply:
Datatype for epochs is ```AoEpochData[] | undefined``` so I think this should be fine.